src/Html.php

```diff
@@ -3472,6 +3472,7 @@ public static function initEditorSystem(
         array $add_body_classes = [],
         string $toolbar_location = 'top',
         bool $init = true,
+        bool $init_on_demand = false,
```

Comment:
Seems dangerous to add a new parameter in the middle, old code implementation might not be using named parameters.

Should be added at the end instead.